### PR TITLE
fix: Propagate tags to pip-compile rule

### PR DESCRIPTION
**Pull request by ewianda**
This can be usefull if the user wants to disable the test

### CODE REVIEW

File: uv/private/pip.bzl [hunk 1]
```diff
@@ -71,7 +71,7 @@ _pip_compile_test = rule(
     test = True,
 )
 
-def pip_compile(name, requirements_in = None, requirements_txt = None, target_compatible_with = None, python_platform = None):
+def pip_compile(name, requirements_in = None, requirements_txt = None, target_compatible_with = None, python_platform = None, **kwargs):
```
Comment (mark-thm) [inline uv/private/pip.bzl:74]: I'd prefer to keep this explicit vs. taking in kwargs, mind updating?

**Review by mark-thm — COMMENTED** [review]
Thanks for your contribution! Happy to accept something in this vein, but we'd prefer to keep the arguments explicit vs. passing around kwargs and dropping unused ones silently.

File: uv/private/pip.bzl (hunk 1 above)
Comment (ewianda) [reply uv/private/pip.bzl]: done

**Review by mark-thm — APPROVED** [review]

**Comment by mark-thm** [comment]
Releasing in v0.6.0.

**Pull request merged**